protected/assets/js/script.js

```diff
@@ -143,28 +143,29 @@ $(document).ready(function(){
 			return;
 		}
 		var messages = warning.find('.messages');
-		var box = $('<div class="quicklook warning"></div>');
-
-		box.hide();
-		box.html(messages.html());
-		box.appendTo('body');
-
-		warning.hover(function() {
-
-			var offsetPos = $(this).offset();
-			var top = offsetPos.top + $(this).height() + 6;
-			var middle = offsetPos.left + $(this).width()/2;
-			var left = middle - box.width()/2 - 8;
-
-			box.css({
-				position: 'absolute',
-				top: top,
-				left: left
-			});
-			box.fadeIn('fast');
-		}, function(e){
-			box.hide();
-		});
+		if (messages.length) {
```

Comment:
You could use the same pattern they use above and just return out of the function if(!messages.length)

Reply:
ok